app/styles/main.scss

```diff
@@ -1,6 +1,43 @@
 //////////////////////////////////////////////////////////////////////////////////////////
 // TOP LEVEL IMPORTS - DO NOT ADD CSS HERE
 
-@import 'utils/all';
```

Comment:
We had multiple layers of imports. Feels much cleaner with one entry point that is responsible for importing all major.